metadata/_category/_template.md

```diff
@@ -0,0 +1,52 @@
+<!-- The filename should match the OpenSGID layer_name. For example, roads.md. The category will be inferred from the directory structure -->
+
+# Title
+
+<!-- The human-readable layer name that should match the pageTitle item in the data page's metadata. If it's a statewide layer, it should include "Utah", e.g. "Utah Roads" -->
+
+## Brief Summary
+
+<!-- A super-short, one-liner description of the dataset. Corresponds to the pageDescription item in the data page's metadata -->
+
+## Summary
+
+<!-- A brief (<2048 characters for AGOL) explanation of the dataset to give the user a high-level overview of what the layer is when skimming lists of datasets. Will be the first description people see in Hub open data and should match the Summary section of the layer's data page  -->
+
+## Description
+
+<!-- A more in-depth explanation of the dataset, where it came from and how its created, so the user can decide if its what they need. It should answer the following questions (but the answers don't need to be formatted in this order; they can be merged, combined, or woven into a narrative if sensible):
+
+What is the dataset
+What is the purpose of the dataset
+What does the dataset represent
+How was the dataset created
+How reliable and accurate is the dataset
+-->
+
+## Credits
+
+<!-- List both who created the original data and who created and is hosting the dataset itself. For example, the roads layer data come from UDOT, cities, and counties and is created and hosted by UGRC. -->
```

Comment:
it would make parsing simpler if these had subheadings. Maybe a description of the situation and some parsable sections?

Reply:
This to me captures the data vs dataset steward and host in the sgid index. The description of the situation is your 

> For example, the roads layer data come from UDOT, cities, and counties and is created and hosted by UGRC.

But parsing out what role UDOT and UGRC have from the description is difficult.

Reply:
Ah, gotcha. Let me see what I can do.